Revert CFB block length change preventing reading older files (i.e.
encfs blocks -
http://sourceforge.net/mailarchive/message.php?msg_name=BAY114-W25D2FAC625E0E4A7D37BA7FE390@phx.gbl
)

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>

--- openssl/crypto/evp/evp_locl.h	2010/02/15 19:40:45	1.10.2.7
+++ openssl/crypto/evp/evp_locl.h	2010/02/26 14:41:38	1.10.2.8
@@ -127,9 +127,9 @@
 #define BLOCK_CIPHER_def_cfb(cname, kstruct, nid, key_len, \
 			     iv_len, cbits, flags, init_key, cleanup, \
 			     set_asn1, get_asn1, ctrl) \
-BLOCK_CIPHER_def1(cname, cfb##cbits, cfb##cbits, CFB, kstruct, nid, \
-			(cbits + 7)/8, key_len, iv_len, \
-		flags, init_key, cleanup, set_asn1, get_asn1, ctrl)
+BLOCK_CIPHER_def1(cname, cfb##cbits, cfb##cbits, CFB, kstruct, nid, 1, \
+		  key_len, iv_len, flags, init_key, cleanup, set_asn1, \
+		  get_asn1, ctrl)
 
 #define BLOCK_CIPHER_def_ofb(cname, kstruct, nid, key_len, \
 			     iv_len, cbits, flags, init_key, cleanup, \
